document/core/valid/modules.rst

```diff
@@ -175,15 +177,19 @@ Element segments :math:`\elem` are classified by :ref:`segment types <syntax-seg
      \qquad
      C \vdashexprconst \expr \const
      \qquad
+     C \vdashexprconst et = \FUNCREF
```

Comment:
There is no C vdash here, since this is just a plain syntactic equality.

Reply:
Done.